eisbuk-admin/public/locales/en/translation.json

```diff
@@ -11,6 +11,7 @@
     "NonReversible": "This action is non-reversible",
     "Slot": "slot",
     "Book": "Book",
+    "Reserved": "Reserved",
     "Booked": "Booked",
```

Comment:
Duplicate, I prefer "Booked" as it's more consistent with the rest of the semantics (filenames and translations)